main.go

```diff
@@ -72,13 +72,13 @@ func main() {
 	data.Logger = logger
 	for _, a := range addresses {
 		logger.Debug("Parsing: ", a)
-		url, err := urlx.Parse(a)
+		my_url, err := urlx.Parse(a)
```

Comment:
that is not my URL, that is user of Prometheus server, why would you like to name it this way?